tgui/packages/tgui/styles/main.scss

```diff
@@ -19,40 +20,9 @@
 @include meta.load-css('./atomic/outline.scss');
 @include meta.load-css('./atomic/text.scss');
 
-// Components
-@include meta.load-css('./components/BlockQuote.scss');
-@include meta.load-css('./components/Button.scss');
-@include meta.load-css('./components/ColorBox.scss');
-@include meta.load-css('./components/Dialog.scss');
-@include meta.load-css('./components/Dimmer.scss');
-@include meta.load-css('./components/Divider.scss');
-@include meta.load-css('./components/Dropdown.scss');
-@include meta.load-css('./components/Flex.scss');
-@include meta.load-css('./components/Icon.scss');
-// |GOONSTATION-CHANGE| This component's stylesheet includes styles that apply to other things,
-// and the component is not used by Goonstation; including here for completeness, and a note that
-// until this is fixed including *all* of tgui-styles may break a bunch of our styles
-// @include meta.load-css('./components/ImageButton.scss');
```

Comment:
Fixed upstream, we do not consume this component currently (but its styles would be included if this was still a problem).